From 87223669ab28303004a66c24ee2aafd770d84c60 Mon Sep 17 00:00:00 2001
From: Jorge <jlamperez10@gmail.com>
Date: Tue, 18 Oct 2016 17:59:47 +0200
Subject: [PATCH 2/8] 	Changed: board/xilinx/zynq/Makefile

---
 board/xilinx/zynq/Makefile | 1 +
 1 file changed, 1 insertion(+)

diff --git a/board/xilinx/zynq/Makefile b/board/xilinx/zynq/Makefile
index eab9303..9f9e0cc 100644
--- a/board/xilinx/zynq/Makefile
+++ b/board/xilinx/zynq/Makefile
@@ -13,6 +13,7 @@ hw-platform-$(CONFIG_TARGET_ZYNQ_MICROZED)	:= MicroZed_hw_platform
 hw-platform-$(CONFIG_TARGET_ZYNQ_ZC702)		:= ZC702_hw_platform
 hw-platform-$(CONFIG_TARGET_ZYNQ_ZC706)		:= ZC706_hw_platform
 hw-platform-$(CONFIG_TARGET_ZYNQ_ZYBO)		:= zybo_hw_platform
+hw-platform-$(CONFIG_TARGET_ZYNQ_ZYNQBERRY)     := zynqberry_hw_platform
 # If you want to use customized ps7_init_gpl.c/h,
 # enable CONFIG_ZYNQ_CUSTOM_INIT and put them into custom_hw_platform/.
 # This line must be placed at the bottom of the list because
-- 
1.9.1

